ddtrace/internal/telemetry/logging.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import os
+import traceback
+
+from ddtrace.internal.telemetry.constants import TELEMETRY_LOG_LEVEL
+
+
+class DDTelemetryLogHandler(logging.Handler):
+    CWD = os.getcwd()
+
+    def __init__(self, telemetry_writer):
+        self.telemetry_writer = telemetry_writer
+        super().__init__()
+
+    def emit(self, record):
+        # type: (logging.LogRecord) -> None
+        if record.levelno >= logging.ERROR:
+            # Capture start up errors
+            full_file_name = os.path.join(record.pathname, record.filename)
+            self.telemetry_writer.add_error(1, record.msg % record.args, full_file_name, record.lineno)
```

Comment:
This duplicates this logic: https://github.com/DataDog/dd-trace-py/blob/ygree/integration-exception-tracking/ddtrace/internal/logger.py#L42-#L46. We should remove lines 42-46 from `logger.py`.